src/format_text.js

```diff
@@ -1,13 +1,13 @@
-import { sprintf } from 'sprintf-js';
+import { sprintf } from 'sprintf-js'
 
 
 // Regexes taken from or inspired by sprintf-js
 const Regex = {
-    TEMPLATE_LITERAL: /\${([^\)]+?)}/g,
+    TEMPLATE_LITERAL: /\${([^)]+?)}/g,
```

Comment:
I suck at regex so need confirmation if removing the escaping here still produces same result